modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache

```diff
@@ -25,6 +29,13 @@ import java.util.Collections;
 import java.util.List;
 import java.util.StringJoiner;
 import java.util.function.Consumer;
+import java.util.Optional;
+import java.util.zip.GZIPInputStream;
```

Comment:
shouldn't this import for `java.util.zip.GZIPInputStream` only needed when useGzipFeature is enabled?